AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/property/control/PropertyValueService.java

```diff
@@ -71,7 +71,7 @@ public void setPropertyValue(ContextDependency<?> resourceContext, Integer prope
         if (!propertyValidationService.canPropertyValueBeSetOnContext(p.getDescriptor(), resourceContext)) {
             throw new ValidationException("The property " + p.getDescriptor() + " can not be set on context " + resourceContext.getContext());
         }
-        p.setValue(unobfuscatedValue);
+        p.setValueAndEncrypt(unobfuscatedValue);
```

Comment:
@yvespp nope, perhaps the method's name is a bit misleading but it only encrypts values if the Descriptor returns true for isEncrypt() (which is true for values which shall be encrypted)